template_example/dev/ansible/03_healthcheck_test.yml

```
@@ -0,0 +1,68 @@
+- name: set variable
+  set_fact:
+    consul_token: "master"
```

The same here with lookup
```suggestion
    consul_token: "{{ lookup('env', 'consul_master_token') }}"
```

---

if i change to this then it will fail as this env variable is available only in 0.6 version of the box.
https://github.com/fredrikhgrelland/vagrant-hashistack-template/blob/7ad8b6a36d29b5da71c179493e553e6f9b4a6fa7/Vagrantfile.default#L3

Reference to the earlier conversation thread 
https://github.com/fredrikhgrelland/vagrant-hashistack-template/pull/63#discussion_r508222525

---

as ok. I am sorry (context switching). We might need to resolve it in next PR. Relevant to https://github.com/fredrikhgrelland/vagrant-hashistack/issues/394